lib/meta-box-partial-page.php

```diff
@@ -550,3 +189,102 @@ function gutenberg_is_meta_box_empty( $meta_boxes, $context, $post_type ) {
 }
 
 add_filter( 'filter_gutenberg_meta_boxes', 'gutenberg_filter_meta_boxes' );
+
+
+
+/**
+ * Renders the WP meta boxes forms.
+ *
+ * @since 1.8.0
+ *
+ * @param string $locations The metaboxes locations to render.
+ */
+function the_gutenberg_metaboxes( $locations = array( 'advanced', 'normal', 'side' ) ) {
+	global $post, $current_screen, $wp_meta_boxes;
+
+	// Handle meta box state.
+	$_original_meta_boxes = $wp_meta_boxes;
+
+	/**
+	 * Fires right before the meta boxes are rendered.
+	 *
+	 * This allows for the filtering of meta box data, that should already be
+	 * present by this point. Do not use as a means of adding meta box data.
+	 *
+	 * By default gutenberg_filter_meta_boxes() is hooked in and can be
+	 * unhooked to restore core meta boxes.
+	 *
+	 * @param array $wp_meta_boxes Global meta box state.
+	 */
+	$wp_meta_boxes = apply_filters( 'filter_gutenberg_meta_boxes', $wp_meta_boxes );
+
```

Comment:
Now that we're loading it inline, we should call `gutenberg_intercept_meta_box_render()` here - it should be done immediately before the `do_meta_boxes()` calls.

I'm inclined to not attach it to `filter_gutenberg_meta_boxes`, to prevent anything from being added to `$wp_meta_boxes` after it runs.

Reply:
I thought the `gutenberg_intercept_meta_box_render` was meant to display a warning in the classic editor while `the_gutenberg_metaboxes` renders the metaboxes in Gutenberg.

Am I missing something?

Anyway, feel free to push any change to this branch

Reply:
Oh, wait. Yeah, I totally wasn't reading it right. 🙂